include/ametsuchi/tx_store/index.h

```diff
@@ -65,7 +72,24 @@ class Index {
    * @param offsets
    * @return index of the first appended element
    */
-  std::size_t batch_append(std::vector<file::offset_t> offsets);
+  std::size_t append_batch(std::vector<file::offset_t> &offsets);
```

Comment:
Add `const` to the parameter (looks like this method doesn't modify the parameter).